modules/lmdbbackend/lmdbbackend.cc

```diff
@@ -716,6 +715,9 @@ LMDBBackend::LMDBBackend(const std::string& suffix)
       throw std::runtime_error(std::string("running with Lightning Stream support requires shards=1"));
     }
   }
+  else {
+    LMDBLS::s_flag_deleted = mustDo("flag-deleted");
```

Comment:
while there's no race on the others (random_ids, handle_dups), we might want to move them too, for clarity?